src/WebJobs.Extensions.DurableTask/Listener/DurableTaskTargetScaler.cs

```diff
@@ -52,8 +52,8 @@ public async Task<TargetScalerResult> GetScaleResultAsync(TargetScalerContext co
                 var serializedControlQueueLengths = metrics.ControlQueueLengths;
                 var controlQueueLengths = JsonConvert.DeserializeObject<IReadOnlyList<int>>(serializedControlQueueLengths);
 
-                var controlQueueMessages = controlQueueLengths.Sum();
-                var activeControlQueues = controlQueueLengths.Count(x => x > 0);
+                var controlQueueMessages = controlQueueLengths!.Sum();
+                var activeControlQueues = controlQueueLengths!.Count(x => x > 0);
```

Comment:
I can only pass the build/release CI after I added this !. Otherwise, it will show me a error that `controQueueLengths` could be null. I am not sure why this is triggered now.

Reply:
I suppose .NET6 might have better null-value detection / nullability analysis